torch/distributed/fsdp/_common_utils.py

```diff
@@ -184,6 +201,47 @@ def _is_fsdp_flattened(tensor: torch.Tensor) -> bool:
     return getattr(tensor, FSDP_FLATTENED, False)
 
 
+def _get_handles_to_fqns(
```

Comment:
Since we are mapping `FlatParameter` to FQN, shall we name this `_get_flat_param_to_fqns`? It looks like above you already named the variable `flat_param_to_fqns`.

Reply:
_get_flat_param_to_fqns is kind of being used in state dict codes.., and it is designed for non-composable path. Since composable path, there is no 'real' flat_param concept, possibly I can change it to be '_get_flat_param_handles_to_fqns()'? and the key could be handle, not flat_param

Reply:
Feel free to disagree, but I feel that even for composable path, there is still the concept of `FlatParameter`, only that it is not exposed to the user.

I think here, the important thing is to have the function name match the `dict`'s keys and values. I think `_get_flat_param_handle_to_fqns()` is fine since that means it maps `flat_param_handle: FlatParamHandle` to `fqns: List[str]`.